File: app/src/main/java/com/example/topmejorestiendas/RegistroActivity.java
package com.example.topmejorestiendas;

import android.content.Intent;
import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import com.example.topmejorestiendas.databinding.ActivityRegistroBinding;

public class RegistroActivity extends AppCompatActivity {
    private ActivityRegistroBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityRegistroBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        binding.btnUsuarioCliente.setOnClickListener(v -> {
            startActivity(new Intent(this, RegistroUsuarioActivity.class));
        });

        binding.btnDuenoNegocio.setOnClickListener(v -> {
            startActivity(new Intent(this, RegistroLocalActivity.class));
        });

        binding.tvBackToLogin.setOnClickListener(v -> finish());
    }
}

File: app/src/main/java/com/example/topmejorestiendas/RegistroActivity.kt
package com.example.topmejorestiendas

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.topmejorestiendas.databinding.ActivityRegistroBinding

class RegistroActivity : AppCompatActivity() {
    private lateinit var binding: ActivityRegistroBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityRegistroBinding.inflate(layoutInflater)
        setContentView(binding.root)

        binding.btnUsuarioCliente.setOnClickListener {
            startActivity(Intent(this, RegistroUsuarioActivity::class.java))
        }

        binding.btnDuenoNegocio.setOnClickListener {
            startActivity(Intent(this, RegistroLocalActivity::class.java))
        }

        binding.tvBackToLogin.setOnClickListener {
            finish()
        }
    }
}